docker-for-mac/release-notes.md

```diff
@@ -22,13 +22,25 @@ Desktop for Mac](install.md#download-docker-for-mac).
 
 ## Stable Releases of 2019
 
-## Docker Desktop Community 2.1.0.2
-2019-09-04
+## Docker Desktop Community 2.1.0.3
+2019-09-16
 
 > [Download](https://hub.docker.com/?overlay=onboarding)
 >
 > You must sign in to Docker Hub to download Docker Desktop.
 
+### Upgrades
+
+- [Kitematic 0.17.8](https://github.com/docker/kitematic/releases/tag/v0.17.8)
+### Bug fixes and minor changes
```

Comment:
Missing new line?

Reply:
This has been fixed. Please see https://github.com/docker/docker.github.io/pull/9450/commits/9428e6c1d36d797d6de09c0fa2d36d2438727d7a